docs/ops/deployment/yarn_setup.md

```diff
@@ -251,6 +250,29 @@ The user-jars position in the class path can be controlled by setting the parame
 - `FIRST`: Adds the jar to the beginning of the system class path.
 - `LAST`: Adds the jar to the end of the system class path.
 
+## Run an application in Application Mode
+
+To launch an application in [Application Mode]({{ site.baseurl }}/ops/deployment/#deployment-modes), you can type:
+
+{% highlight bash %}
+./bin/flink run-application -t yarn-application ./examples/batch/WordCount.jar
+{% endhighlight %}
+
+The command above, goes against the recently introduced "Generic CLI". So, apart from the `-t`, all 
```

Comment:
I think we shouldn't mention `Generic CLI` here and that it was recently introduced. This will not age well. 😅

But it's good to describe that basically the only custom parameter is `-t`, everything else is as in the config.